### DISCUSSION TITLE
Official source code repo for EJBCA CE

### DISCUSSION THREAD

**Discussion by fco-code-org** [Keyfactor/ejbca-ce#697] [General]
Hello,

What is the official repo for EJBCA CE source code ?
Is it Github or Source Forge ? 

Best Regards.

**Comment by mike-agrenius-kushner** [comment]
Hi, SourceForge was deprecated years ago, and is only being updated with releases for historical reasons. 

GitHub is where we publish the latest CE mirror of the code.

Cheers,
Mike
